hw2/dbFunctions.js

```diff
@@ -0,0 +1,20 @@
+const { v4: uuidv4 } = require("uuid");

```

Comment:
dbFunctions is a bad naming, create module(subfolder) "users", with controllers for each route and UserModel in which you can put your dbFunctions, it's a common practice of structuring your files known as MVC
Model - View - Contorller
(we don't have view)